src/hooks/subscription/CommunityUpdatesNotifier.tsx

```diff
@@ -0,0 +1,48 @@
+import { useApolloErrorHandler, useConfig, useNotification, useUserContext } from '..';
+import { useCommunicationUpdateMessageReceivedSubscription } from '../generated/graphql';
+import { FEATURE_COMMUNICATIONS, FEATURE_SUBSCRIPTIONS } from '../../models/constants';
+import { logger } from '../../services/logging/winston/logger';
+
+const useCommunityUpdatesNotifier = () => {
+  const { isFeatureEnabled } = useConfig();
+  const { isAuthenticated } = useUserContext();
+
+  if (!isFeatureEnabled(FEATURE_COMMUNICATIONS) || !isFeatureEnabled(FEATURE_SUBSCRIPTIONS) || !isAuthenticated) {
+    return;
+  }
```

Comment:
Broken Rule of hooks 
Hooks can not be called conditionally.
https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level
![image](https://user-images.githubusercontent.com/3524687/148751774-7321727b-154e-4bff-816e-27654582f3a7.png)